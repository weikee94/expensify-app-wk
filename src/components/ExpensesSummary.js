// stateless functional component
import React from 'react';
import { connect } from 'react-redux';
import numeral from 'numeral';