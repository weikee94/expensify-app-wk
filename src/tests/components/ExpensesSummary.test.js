import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpensesSummary';

test('should correctly render expense summary with 1 expense', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={1} expensesTotal={230}/>);
    expect(wrapper).toMatchSnapshot();
});

test('should correctly render expense summary with multiple expenses', () => {
    const wrapper = shallow(<ExpenseSummary expenseCount={33} expensesTotal={23012304567}/>);
    expect(wrapper).toMatchSnapshot();
});