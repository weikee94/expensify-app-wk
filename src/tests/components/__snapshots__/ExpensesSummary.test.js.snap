// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expense summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.30
  </h1>
</div>
`;

exports[`should correctly render expense summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    33
     
    expenses
     totalling 
    $230,123,045.67
  </h1>
</div>
`;
